### ISSUE TITLE
Hidden calendar, API calls should silently fail

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2770]
Originally reported on Google Code with ID 2505

```
Description: 

A calendar is created inside a container div with display:none. Calling the select
method on the calendar causes the js error 'TypeError: offsets is null'. Setting the
container's display to block, and calling render on the calendar eliminates the error
on subsequent calls to select.

Steps to reproduce:

See https://jsfiddle.net/j8u9bzdL/

The example sets up a calendar inside a hidden div, along with two buttons. The 'Toggle'
button shows/hides the container div, and renders the calendar. The 'Select' button
calls the calendar's select method with a pair of dates.

1. Click 'Select'. Note the js error 'TypeError: offsets is null' in the console window.
2. Click 'Toggle'. The calendar is now shown.
3. Click 'Select'. Note that there is no error in the console window.
4. Click 'Toggle'. The calendar is now hidden.
5. Click 'Select'. Note that there is no error in the console window.

All subsequent calls to select will succeed, regardless of whether the calendar is
hidden or not.

```

Reported by `silktw` on 2015-05-13 20:47:13

Imported with **1** stars.

**Comment by arshaw** [comment]
this is fixed in v4 e481bf3981aa019eda91417c8196b100cda15c28

will comment again when v4 is officially released

**Comment by arshaw** [comment]
[V4 was officially released a few days ago!](https://fullcalendar.io/blog/2019/03/v4-officially-released)
